src/Sylius/Behat/Context/Api/Admin/ManagingTaxCategoriesContext.php

```diff
@@ -23,17 +24,22 @@ final class ManagingTaxCategoriesContext implements Context
     /** @var ApiClientInterface */
     private $client;
 
-    public function __construct(ApiClientInterface $client)
+    /** @var ResponseCheckerInterface */
+    private $responseChecker;
+
+    public function __construct(ApiClientInterface $client, ResponseCheckerInterface $responseChecker)
     {
         $this->client = $client;
+        $this->client->setResource('tax_categories');
```

Comment:
I would prefer to create clients per request instead of setting it in the constructor. Right now, class API wouldn't be obvious for me. It is not possible to use this service without this configuration, still, it is super easy to omit this.

Reply:
That's the next step I would like to do 👌

Reply:
See [this discusion](https://github.com/Sylius/Sylius/pull/11210/files#r391062258)